src/Microsoft.Azure.WebJobs.Host/Executors/FunctionExecutor.cs

```diff
@@ -143,8 +143,7 @@ public async Task<IDelayedException> TryExecuteAsync(IFunctionInstance functionI
             }
             finally
             {
-                var funcEnd = Stopwatch.GetTimestamp();
```

Comment:
Are you sync'ed to current dev branch? The code [here](https://github.com/Azure/azure-webjobs-sdk/blob/bba8d6a69f77f3fea870082efcd540a138a788b4/src/Microsoft.Azure.WebJobs.Host/Executors/FunctionExecutor.cs#L147) isn't the same as in your PR?

Reply:
Also, there are several places in the execution pipeline where we're calling StopWatch.StartNew and/or creating per invocation StopWatches. Is there really a perf issue with using SW? If so, then I think we'd want to discuss a consistent replacement pattern to use consistently throughout.

Reply:
I updated to latest (was a bit behind) - yep the Stopwatches add up overall enough to make an impact on RPS. I agree we should look at replacements overall  - wanted to put this forth as the location more than the fix.

A common pattern has been to include `ValueStopwatch` as an internal copy from the framework itself (e.g. https://github.com/Azure/azure-sdk-for-net/blob/4162f6fa2445b2127468b9cfd080f01c9da88eba/sdk/core/Azure.Core/src/Shared/ValueStopwatch.cs) - this gives us a consistent usage throughout, but making it internal would means `InternalsVisibleTo`. The pattern we have elsewhere is `[Obsolete("Not ready for public consumption.")]` - could go that way for consistency too.

Thoughts on including `ValueStopwatch` and doing one of the above for a consistent pattern? I could go through and replace all references that way.